### ISSUE TITLE
Check if the literals are null

### ISSUE THREAD

**Issue by mosoriob** [tirthmehta/WINGS_WORKFLOW_PROVENANCE_SCENARIOS#23]
**Describe the bug**

The code generates many null exceptions
```
        if(rs.hasNext()){
            //variables to extract: ?doc ?contrib ?time ?license
            QuerySolution qs = rs.next();
            try{
                Literal docContent = qs.getLiteral("?doc");
                wtInstance.addLiteral(opmwModel.createProperty(Constants.OPMW_DATA_PROP_HAS_DOCUMENTATION), docContent);
            }catch(Exception e){}
            try{
                Literal contrib = qs.getLiteral("?contrib");
                OntClass agentClass = opmwModel.createClass(Constants.OPM_AGENT);
                Individual contributor = agentClass.createIndividual(Constants.PREFIX_EXPORT_RESOURCE+
                        Constants.CONCEPT_AGENT+"/"+URLEncoder.encode(""+contrib, "UTF-8"));
                contributor.addLabel(contrib);
                wtInstance.addProperty(opmwModel.createProperty(Constants.PROP_HAS_CONTRIBUTOR), contributor);
            }catch(Exception e){}
            try{
                Literal timeLastModified = qs.getLiteral("?time");
                wtInstance.addLiteral(opmwModel.createProperty(Constants.DATA_PROP_MODIFIED), timeLastModified);
            }catch(Exception e){}
            try{
                Literal userVersion = qs.getLiteral("?version");
                wtInstance.addLiteral(opmwModel.createProperty(Constants.OPMW_DATA_PROP_RELEASE_VERSION), userVersion);
            }catch(Exception e){}
            try{
                Literal license = qs.getLiteral("?license");
                wtInstance.addLiteral(opmwModel.createProperty(Constants.DATA_PROP_RIGHTS), license);
            }catch(Exception e){//no license declared, add license by default
                opmwModel.add(wtInstance,opmwModel.createProperty(Constants.DC_LICENSE),"http://creativecommons.org/licenses/by/3.0/",
                        XSDDatatype.XSDanyURI);
            }
        }
```
**To Reproduce**
1. Export a execution

**Expected behavior**
Check if the literals are null

**Comment by dgarijo** [comment]
@sirspock which URI has null literals? The code above seems to generate everything correctly.

**Comment by mosoriob** [comment]
The screenshot shows an example.
We dont have information about the time and license and it will generate a null exception.

![Screen Shot 2019-06-18 at 2 52 48 PM](https://user-images.githubusercontent.com/2365990/59725957-bae78800-91fd-11e9-8299-7cf78a9c550f.png)

**Comment by dgarijo** [comment]
Ohh, I see, there is no exception, I just needed to check for nulls. It's an easy fix

**Comment by mosoriob** [comment]
Fixed at ce9576f95c9f63b037ccd0aa344c818735725d76
